assets/sass/utilities/variables/_color-var.scss

```diff
@@ -0,0 +1,316 @@
+//--------------------------------------------------------------
```

Comment:
Some general comments here, rather than noting in a handful of comments on individual lines.

Sometimes the color variables are set like `$color-background-element` and other times `$color-element-background`. Sometimes variables use `color` twice, like `$color-element-color`.

I wonder if we can make this more consistent so everything is always `$color-background-element` for backgrounds and `$color-element` for colors?

Some examples from in this file:
`$color-background-screen-reader`
`$color-text-screen-reader`

I like those!

Then we have some others like this:
`$color-icon-close-background`
`$color-modal-close-color`

Those feel a little clunkier to me, especially the second one where `color` is used twice.

More than happy to compromise here if these were done a specific way for specific reasons, but just something I noticed that didn't seem to be done consistently.

Reply:
Good call. Should keep these consistent for sure.